core/src/main/resources/hudson/model/Job/buildTimeTrend_pt_BR.properties

```diff
@@ -22,7 +22,7 @@
 
 title={0} Tend\u00eancia de tempo de builds
 Duration=Dura\u00e7\u00e3o
-Slave=SlaveMore\ than\ 1\ builds\ are\ needed\ for\ the\ trend\ report.=\u00c9 necess\u00e1rio mais do que um build para criar o relat\u00f3rio de tend\u00eancia.
+Agent=SlaveMore\ than\ 1\ builds\ are\ needed\ for\ the\ trend\ report.=\u00c9 necess\u00e1rio mais do que um build para criar o relat\u00f3rio de tend\u00eancia.
```

Comment:
It was one line before this change, I'm sure that if I would have split this line you would have put a bug for unrelated changes :smiley_cat: